tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/ConsumerGroupHelper.cs

```diff
@@ -13,19 +13,30 @@ internal class ConsumerGroupHelper
 {
     // A map between Kafka Consumer<TKey,TValue> and the corresponding consumer group
     private static readonly ConditionalWeakTable<object, string> ConsumerToGroupIdMap = new();
+    private static readonly ConditionalWeakTable<object, string> ConsumerToBootstrapServersMap = new();
```

Comment:
I thought I responded to this, sorry!

I assumed that creating a new `Tuple<string, string>` object would be worse for performance because this would add a new allocation for each new tuple, whereas using the existing strings would not require additional allocations